lib/linguist/language.rb

```diff
@@ -499,12 +499,12 @@ def inspect
     end
   end
 
-  extensions = Samples.cache['extnames']
+  extensions   = Samples.cache['extnames']
   interpreters = Samples.cache['interpreters']
-  filenames = Samples.cache['filenames']
-  popular = YAML.load_file(File.expand_path("../popular.yml", __FILE__))
+  filenames    = Samples.cache['filenames']
```

Comment:
This is safe to remove. I forgot to do it in 0bf4b8a4826356a882c2baca16722991aa4c4b0c.

Reply:
Perfect! Works for me. :grinning: Consider it gone, and my drastic spike of Ruby fluency to be concerning

(for me)

Reply:
And don't worry, I *did* double-check the `||=` operator to guarantee it did [what I thought it does](http://www.rubyinside.com/what-rubys-double-pipe-or-equals-really-does-5488.html). :wilted_flower:

There's no good reason I should be this good at Ruby, I'm always bitching about why its ecosystem are the antichrist :cry: